src/NLog/LogEventInfo.cs

```diff
@@ -551,12 +551,12 @@ public void SetStackTrace(StackTrace stackTrace, int userStackFrame)
         /// Sets the details retrieved from the Caller Information Attributes
         /// </summary>
         /// <param name="callerClassName"></param>
-        /// <param name="callerMemberName"></param>
+        /// <param name="callerMethodName"></param>
         /// <param name="callerFilePath"></param>
         /// <param name="callerLineNumber"></param>
-        public void SetCallerInfo(string callerClassName, string callerMemberName, string callerFilePath, int callerLineNumber)
+        public void SetCallerInfo(string callerClassName, string callerMethodName, string callerFilePath, int callerLineNumber)
```

Comment:
I'm afraid this is confusing. Please membername everywhere. Using 2 names for one thing is confusing.

Reply:
Fixed

Reply:
Is still see many callerMethodNames?

Reply:
Thought you agreed that `callerMemberName` was a confusing name for something that is a method-name. See no reason for a badly chosen property-name to obfuscate the rest of the internal code (But leaving the smell in the public interface). Notice that when extracting the method-name from stacktrace, then it is correctly called method-name.